Gems/LmbrCentral/Code/Source/Shape/EditorPolygonPrismShapeComponentMode.cpp

```diff
@@ -27,7 +29,7 @@ namespace LmbrCentral
         AzToolsFramework::MidpointCalculator midpointCalculator;
         for (const AZ::Vector2& vertex : vertexContainer.GetVertices())
         {
-            midpointCalculator.AddPosition(AZ::Vector2ToVector3(vertex, height));
+            midpointCalculator.AddPosition(AZ::Vector3(vertex, height));
```

Comment:
there is a hidden dependency through VertexContainerInterface.h ...

Reply:
Ah good spot, is this fixed now? Shall I kick off another build?

Reply:
Nice, PR is green, will merge first thing tomorrow :+1:

Reply:
I'll merge it now.

Reply:
Thank you! 🙂